src/support/libinfo.cc

```diff
@@ -24,11 +24,230 @@
 #define TVM_GIT_COMMIT_HASH "NOT-FOUND"
 #endif
 
+#ifndef TVM_USE_CUDA
+#define TVM_USE_CUDA "NOT-FOUND"
+#endif
+
+#ifndef TVM_USE_OPENCL
```

Comment:
Let us change the macro prefix, as TVM_USE_XYZ is used for normal command switch. Instead, let us use TVM_INFO_USE_OPENCL

Reply:
It makes sense. Fixed. Thank you!